README.rst

```diff
@@ -56,9 +56,17 @@ The SkyPy library can then be imported from python:
     >>> import skypy
     >>> help(skypy)
 
+SkyPy has a number of optional dependencies which can be installed separately.
+One of these is `skypy-data`_ which contains generic data (such as Johnson bandpasses)
+used for calculations in SkyPy. This can be installed with:
```

Comment:
```suggestion
One of these is `skypy-data`_ which contains data such as photometric bandpasses
required for some calculations in SkyPy. This can be installed with:
```